packages/reactivity/src/ref.ts

```diff
@@ -3,7 +3,7 @@ import { OperationTypes } from './operations'
 import { isObject } from '@vue/shared'
 import { reactive } from './reactive'
 
-export const refSymbol = Symbol(__DEV__ ? 'refSymbol' : undefined)
+export const refSymbol = Symbol(__DEV__ ? 'refSymbol' : void 0)
```

Comment:
This is even shorter and consistent with other symbols in the codebase.

```suggestion
export const refSymbol = Symbol(__DEV__ ? 'refSymbol' : '')
```